flutter/lib/common/widgets/toolbar.dart

```diff
@@ -403,7 +406,7 @@ Future<List<TRadioMenu<String>>> toolbarImageQuality(
       groupValue: groupValue,
       onChanged: (value) {
         onChanged(value);
-        customImageQualityDialog(ffi.sessionId, id, ffi);
+        customImageQualityDialog(ffi.sessionId, id, ffi, customFpsRange, customQualityRange);
```

Comment:
`customImageQualityDialog(SessionID sessionId, String id, FFI ffi, NumRange customQualityRange, NumRange customFpsRange)`

The order of the parameters needs to be adjusted.